src/state/connection-mixin.ts

```diff
@@ -156,8 +157,25 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
 
     protected hassReconnected() {
       super.hassReconnected();
+
       this._updateHass({ connected: true });
       broadcastConnectionStatus("connected");
+
+      // @ts-ignore
+      this.hass!.callWS({ type: "get_config" }).then((config: HassConfig) => {
+        this._updateHass({ config });
+        if (config.state !== "RUNNING") {
+          const unsubProm = this.hass!.connection.subscribeEvents(() => {
```

Comment:
Careful here. If Home Assistant finished starting before the subscription was in place, you will never know ! So you will need to fetch the config one more time once `unsubProm` resolves.